antenna-documentation/src/site/markdown/sw360-data-model.md.vm

```diff
@@ -76,4 +76,8 @@ In the following section there is an outline what data of SW360 ${docNameCap} ma
   |-----------------------|-------------------------------|
   | Source File | A zipped folder with the sources of the component. Typically uploaded by the SW360 Uploader |
   | Binary | A known binary of the component, stores also e.g. the hash that can be used to search for the metadata based on a hash. Typically uploaded by the SW360 Uploader |
-  | Clearing Report | A file with curated data and the approval state. If the status of this attachment is `Accepted`, the Clearing State of the component release is `Approved`. Typically uploaded by compliance office tooling. |
\ No newline at end of file
+  | Clearing Report | A file with curated data and the approval state. If the status of this attachment is `Accepted`, the Clearing State of the component release is `Approved`. Typically uploaded by compliance office tooling. |
```

Comment:
Commit message: Please make the reference to the issue go last (between body and footer) and use one of the [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).